Modules/Quotas/src/Quotas.Domain/Aggregates/Identities/ExhaustionDate.cs

```diff
@@ -33,8 +33,8 @@ public override string ToString()
         return left.Value > right.Value;
     }
 
-    public int CompareTo(ExhaustionDate other)
+    public int CompareTo(ExhaustionDate? other)
```

Comment:
This change breaks the implementation of the interface. According to the interface, the parameter of the `CompareTo` method must not be nullable

Reply:
Ok, missed that one - noted.